Tiltfile

```diff
@@ -0,0 +1,10 @@
+allow_k8s_contexts('acn-dev-azure-cni')
+default_registry('ttl.sh/nitishm-12390')
```

Comment:
if this is an ephemeral image can we put it on acnpublic instead?

Reply:
I couldnt find how to provide the credentials (plus we don't want the credentials checked in). Need to look into this